python_modules/libraries/dagster-openai/dagster_openai/resources.py

```diff
@@ -212,7 +215,14 @@ def _wrap_with_usage_metadata(
 
     def setup_for_execution(self, context: InitResourceContext) -> None:
         # Set up an OpenAI client based on the API key.
-        self._client = Client(api_key=self.api_key)
+        kwargs = {}
+        if self.organization:
+            kwargs["organization"] = self.organization
+        if self.project:
+            kwargs["project"] = self.project
+        if self.base_url:
+            kwargs["base_url"] = self.base_url
+        self._client = Client(api_key=self.api_key, **kwargs)
```

Comment:
Could we pass the values directly through parameters instead of using kwargs? These are parameters expected  by the `openai.Client` constructor and can be set to None.

Reply:
Updated in https://github.com/dagster-io/dagster/pull/23260/commits/bea7a4a96ecd3077cf35a9b899cd82970d98350c ✅